### ISSUE TITLE
Disable wandb for command lines?

### ISSUE THREAD

**Issue by CunliangGeng** [MycoAI/MycoAI#2]
When running command lines like `mycoai-classify`, the wandb keeps asking what option I would like to choose (see figure below). However, I don't need wandb at all. Also I don't think it's necessary to use it for end users. It is more useful for the training stage. So I would suggest disabling wandb from the command lines.

![image](https://github.com/user-attachments/assets/3e27393f-19c0-4151-89f9-68a2a9e74bd1)

**Comment by luukromeijn** [comment]
You're right, wandb is not that useful for end users and it complicates the classification procedure a bit. This requires some work.
